tutorials/datasets_tutorial.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import piton
+import piton.datasets
+import contextlib
+import datetime
+import os
+
+# This tutorial covers the two major dataset types in piton, EventCollection
+# PatientCollection
+
+target_directory = "dataset_tutorial_target"
+
+if os.path.exists(target_directory):
+    print("This demo expects an empty target directory")
+    exit(-1)
+else:
+    os.mkdir(target_directory)
```

Comment:
```suggestion
import shutil
shutil.rmtree(target_directory)
os.makedirs(target_directory)
```
(annoying to remember to have to delete this directory each time the script is run)